java/INSTALL

```diff
@@ -103,18 +108,25 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
+     
+     
+  3. Find the full path to JDK
+  
+     Linix    => export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")
```

Comment:
typo: Linux

Should we put some of this logic in the configure script?  That way a user could just have "--java" and it would be located based on finding javac in the path?  That could work for all platforms.  If the user supplies "--java=/path" it would override the automatic detection.